configuration/concepts/breast_examination.csv

```diff
@@ -0,0 +1,23 @@
+Uuid,Void/Retire,Same as mappings,Fully specified name:en,Short name:en,Fully specified name:es,Short name:es,Description:en,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high,_version:1,_order:1800
+0c510fac-1318-4e27-8eab-27cbb3a548ae,,,Results,Results,Resultados,Resultados,,Misc,Text,,,,,,,,,,,
+e3a3ff33-583f-4a4e-b2ab-3e95ccbb24f2,,,Cyclical,Cyclical,Cíclico,Cíclico,,Misc,N/A,,,,,,,,,,,
+e21bc895-e3c5-429f-85cb-2c9524edd0fa,,,Non Cyclical,Non-Cyclical,No cíclico,No cíclico,,Misc,N/A,,,,,,,,,,,
+c23d4502-1798-42ad-9cb6-e35d94e782a9,,,Presence of Breast Lump,Presence of Breast Lump,Presencia de bulto en el seno,Presencia de bulto en el seno,,Misc,Coded,,Yes; No,,,,,,,,,
+d2f7989a-73c4-46ee-a609-5c5da9393d50,,,Presence of Breast Pain,Presence of Breast Pain,Presencia de dolor de mama,Presencia de dolor de mama,,Misc,Coded,,Yes; No,,,,,,,,,
+57dfa129-bcfe-456e-9ef6-5c53925becad,,,Nature of Breast Pain,Nature of Breast Pain,Naturaleza del dolor de mama,Naturaleza del dolor de mama,,Misc,Coded,,Cyclical; Non-Cyclical,,,,,,,,,
+77e70e54-f636-4f3c-b41a-8ea923d0ee35,,,Nipple Discharge,Nipple Discharge,Secreción del pezón,Secreción del pezón,,Misc,Coded,,Yes; No,,,,,,,,,
+de1cfa19-364b-4ed7-bef8-017ae68143f5,,,Inversion,Inversion,inversión,inversión,,Misc,Coded,,Yes; No,,,,,,,,,
+3e900c6c-964e-42b7-b4fe-d24ca2fa729e,,,Rash,Rash,Erupción,Erupción,,Misc,Coded,,Yes; No,,,,,,,,,
+4f9af643-0697-47cb-952f-6b2e29482747,,,Nipple Abnormalities,Nipple Abnormalities,Anormalidades del pezón,Anormalidades del pezón,,ConvSet,N/A,,,Nipple Discharge; Inversion; Rash,,,,,,,,
+152f3d79-7676-46f6-a27b-4212e540dea5,,,Oral Contraceptive Pills Use,Oral Contraceptive Pills,Píldoras anticonceptivas orales,Píldoras anticonceptivas orales,,Misc,Coded,,Yes; No,,,,,,,,,
+01edf3d3-a3dc-4174-9847-dfb5ffdbef9d,,,Hormone Replacement Therapy:,Hormone Replacement Therapy:,Terapia de reemplazamiento de hormonas:,Terapia de reemplazamiento de hormonas:,,Misc,Coded,,Yes; No,,,,,,,,,
+adca267e-a622-4b2d-bd92-e68dad4abdc6,,,History of Breast Issues,History of Breast Issues,Historia de problemas de mama,Historia de problemas  mama,,Misc,Coded,,Yes; No,,,,,,,,,
+db4ce47b-052d-4c90-892f-bb7f13115e8e,,,Seen previously by MD,Seen previously by MD,Visto anteriormente por MD,Visto anteriormente po MD,,Misc,Coded,,Yes; No,,,,,,,,,
+ff0db523-f7b5-4d36-95cf-a25a74672fc0,,,Diagnosis,Diagnosis,Diagnóstico,Diagnóstic,,Misc,Text,,,,,,,,,,,
+f5b35153-76f7-468c-bc39-f44c942f154b,,,Previous Treatment,Previous Treatment,Tratamiento previo,Tratamiento prevo,,Misc,Text,,,,,,,,,,,
+fd663c44-a73a-4dfa-ac1f-655281b09510,,,Imaging,Imaging,Imágenes,Imágenes,,ConvSet,N/A,,,Previous Imaging;Type of Breast Imaging;Date;Results,,,,,,,,
```

Comment:
@basrem could you use the uuid of 'Results' here instead of the FSN otherwise a different concept is being added to the set.

Reply:
Done